packages/admin-ui/src/form-group/form-group.tsx

```diff
@@ -0,0 +1,83 @@
+import type { ReactNode } from 'react'
+import React, { Fragment } from 'react'
+import { createComponent } from '@vtex/admin-ui-react'
+
+import { messages } from './form-group.i18n'
+import { Text } from '../components/Text'
+import { Stack } from '../stack'
+import { Message } from './message'
+import { useMessageFormatter } from '../i18n'
+
+export const FormGroup = createComponent<'div', any>((props) => {
```

Comment:
In my mind, this would be a compound component, that we would use as "lego pieces". Also would change the name for "FormControl". For example: 

```jsx
<FromControl error={/** true or false, provide it using context */} >
  <FromControlLabel>Label</FormGroupLabel>
 
 {/** any other things you may need */}

  <FromControlMessage />
</FromControl>
```

This approach would be better because it stops the implementation to grow over time if we meet different specs in specific components. 

What do you think?